Assets/Mirror/Runtime/NetworkServer.cs

```diff
@@ -843,14 +854,14 @@ public static void SetClientNotReady(NetworkConnection conn)
         }
 
         // default ready handler.
-        static void OnClientReadyMessage(NetworkConnection conn, ReadyMessage msg)
+        void OnClientReadyMessage(NetworkConnection conn, ReadyMessage msg)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this unused method parameter 'msg'.](https://app.codacy.com/manual/paulpach/MirrorNG/pullRequest?prid=4841467)